src/bin/lpython.cpp

```diff
@@ -1658,7 +1658,9 @@ int main(int argc, char *argv[])
                     if (0 < err && err < 256) {
                         return err;
                     } else {
-                        return 1;
+                        // https://stackoverflow.com/a/27117435/15913193
+                        // https://linux.die.net/man/3/system
+                        return WEXITSTATUS(err);
```

Comment:
This does not compile on Windows:
```
D:\a\lpython\lpython\lpython-0.8.0-14-gb2d00ab00\src\bin\lpython.cpp(1663): error C3861: 'WEXITSTATUS': identifier not found
```

Reply:
I'm Sorry, @certik. I forgot the suggestions you gave on this.
Would this: 0e9995174cc1e3cdf8b4363f630f27414dc22fe5 be a valid fix